src/structures/ApplicationCommand.js

```diff
@@ -148,23 +147,24 @@ class ApplicationCommand extends Base {
 
   /**
    * Fetches the permissions for this command.
-   * <warn>This is only available for guild application commands.</warn>
+   * <warn>You must specify guildID on global commands and commands on uncached guilds, otherwise it is ignored.</warn>
+   * @param {Snowflake} [guildID] ID for the guild to fetch permissions for if this is a global command
```

Comment:
I don't think you can get a global command / command for an arbitrary (potentially uncached) guild without the `ApplicationCommandManager`, right?

```suggestion
   * <warn>Required if this application command is handled by a {@link ApplicationCommandManager}, otherwise ignored.</warn>
   * @param {Snowflake} [guildID] ID for the guild to fetch permissions for
```

If this suggestion is more confusing or maybe not as precise, the current change can probably stay.
The `uncached` part just irks me a bit. 😅 Maybe it could be replaced with `arbitrary`?